plugins/tedge_apt_plugin/src/main.rs

```diff
@@ -70,6 +73,17 @@ struct SoftwareModuleUpdate {
     pub path: Option<String>,
 }
 
+#[derive(Debug, Deserialize)]
+struct TedgeConfig {
+    pub apt: AptConfig,
+}
+
+#[derive(Debug, Deserialize)]
+struct AptConfig {
+    pub name: Option<String>,
+    pub maintainer: Option<String>,
+}
```

Comment:
That's perfect. We only need these data from the configuration file and we don't need to use tedge-config to get those.